packages/core/test/tracker.test.ts

```diff
@@ -134,4 +139,28 @@ describe("Tracker", () => {
       });
     });
   });
+
+  describe("when session is not sampled", () => {
+    beforeEach(() => {
+      global.navigator.sendBeacon = jest.fn().mockImplementation();
```

Comment:
nitpick: could you remove `sendBeacon` mocking code? We no longer use sendBeacon.